homework/homework_part_2/JS/app.js

```diff
@@ -0,0 +1,105 @@
+console.log("hello");
+//-------------PART 1: TERMS------------------//
+// What is the difference between interpolation and concatenation? Give an example of each.
+// -- concatenation is when you use + to combine strings together. interpolation is when you use ${string} to combine strings.
+//------------------------------------------------------------//
+// What does the acronym DRY stand for? Why should we pay attention to it? What programming tools have we learned to write DRY code?
+//  -- DRY = DON'T REPEAT YOURSELF. Writing loops instead of repeating yourself. Tab Auto Complete.
+//------------------------------------------------------------//
+// What is the difference between declaring a variable and assigning a value to a variable? What do we mean when we say "define" a variable?
+// -- declaring a variable is giving the variable a name. Assigning a value to a variable is when you give a variable a; number, string or boolean.
+//------------------------------------------------------------//
+// When should we use const and when should we use let?
+// use const when you're declaring a variable that isn't going to chance and use let for any other circumstance
+//------------------------------------------------------------//
+// Unix question: What is a "parent" directory?
+
+// -- it's the directory above the current directory you're working in.
+//------------------------------------------------------------//
+// Unix, again: If you're not sure about how to use a Unix command, how can you get more info without being connected to the internet?
+// 
+//------------------------------------------------------------//
+// More Unix: What is "tab completion" and why is it aweseome?
+
+// -- tab completion is when you're writing code and you hit TAB and it autofinishes what it thinks you wanted to write out
+
+//------------------------------------------------------------//
+
+//-------------PART 2: BOOLEAN EXPRESSIONS AND OPERATORS ... and varable assignment------------------//
+
+  const a = 4;
+  const b = 53;
+  const c = 57;
+  const d = 16;
+  const e = 'Kevin';
+
+//1. a __ b;
+console.log(a < b);
+//2. c __ d; -- 
+console.log(c > d);
+//3. 'Name' __ 'Name';
+console.log('Name' === 'Name');
+//4. a __ b __ c;
+console.log(a < b < c); 
```

Comment:
this is a very common answer students give, and it happens to print `true` to the console, but the expression is not being evaluated the way you likely thought it was.  `a < b < c` is not good JS syntax.  look at the values of a, b, and c, and see if you can think of a less odd answer